test/utils/layout_utils.cpp

```diff
@@ -232,3 +232,109 @@ TEMPLATE_TEST_CASE("Convert SiQAD layout to cube::coord_t coordinate layout", "[
         CHECK(lyt_transformed.get_cell_name({5, 6}) == "output cell");
     }
 }
+
+TEMPLATE_TEST_CASE("Generate random offset::ucoord_t coordinate", "[layout-utils]", (offset::ucoord_t))
```

Comment:
```suggestion
TEMPLATE_TEST_CASE("Generate random coordinate", "[layout-utils]", (offset::ucoord_t, cube::coord_t, siqad::coord_t))
```

Reply:
If that is the case, then this `TEST_CASE` does not need to be a templated one, right? But then also, how are the other coordinates tested?